ui/src/journals/components/JournalPapers.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useMemo } from 'react';
+
+import LiteratureSearchContainer from '../../literature/containers/LiteratureSearchContainer';
+import { JOURNAL_PAPERS_NS } from '../../search/constants';
+
+export const JournalPapers = ({ journalName }: { journalName: string }) => {
+  const baseQuery = useMemo(
+    () => ({
+      q: `publication_info.journal_title.raw:"${journalName}"`,
+    }),
+    [journalName]
+  );
+
+  return (
+    <LiteratureSearchContainer
+      namespace={JOURNAL_PAPERS_NS}
+      baseQuery={baseQuery}
+      noResultsTitle="0 Journal papers"
```

Comment:
is it the same style of info as in other embedded searches?

Reply:
It's the same as experiments, institution and conference embedded search results